lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -138,8 +138,8 @@ const mapStateToProps = (state) => {
           ? '#' + route.color
           : route?.color || '#5A5A5A'
       // Try to populate this attribute, which is required for the vehicle popup to appear.
-      vehicle.routeShortName =
-        vehicle.routeShortName || route?.shortName || true
+      vehicle.routeShortName = vehicle.routeShortName || route?.shortName
+      vehicle.routeLongName = vehicle.routeLongName || route?.longName
```

Comment:
```suggestion
      vehicle.routeLongName = vehicle?.routeLongName || route?.longName
```
Should this be optional?